composer.json

```diff
@@ -17,16 +17,16 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "symfony/monolog-bridge": "^5.4 || ^6.0 || ^7.0",
-        "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
+        "monolog/monolog": "^1.25.1 || ^2.0 || ^3.0",
         "symfony/config": "^5.4 || ^6.0 || ^7.0",
+        "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
         "symfony/http-kernel": "^5.4 || ^6.0 || ^7.0",
-        "monolog/monolog": "^1.25.1 || ^2.0 || ^3.0"
+        "symfony/monolog-bridge": "^5.4 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "symfony/yaml": "^5.4 || ^6.0 || ^7.0",
         "symfony/console": "^5.4 || ^6.0 || ^7.0",
-        "symfony/phpunit-bridge": "^7.1"
+        "symfony/phpunit-bridge": "^7.1",
+        "symfony/yaml": "^5.4 || ^6.0 || ^7.0"
```

Comment:
Not really an issue, but I figure it can't hurt to have the dependencies sorted.